MaxText/monitoring_api.py

```diff
@@ -0,0 +1,182 @@
+# pylint: disable=unused-argument, no-name-in-module
```

Comment:
It really feels like this file is in the wrong project. I wonder if Branden/Surbhi could recommend a better home for it?